### PR TITLE
chore(lint): Add markdown lint

### PR DESCRIPTION
**Pull request by erezrokah**
Lints our markdown files

### CODE REVIEW

**Review by yevgenypats — APPROVED** [review]

File: docs/developers/sdk/table/overview.md [hunk 1]
```diff
@@ -4,26 +4,26 @@ Table is the main building block in the SDK provider schema, these tables are pa
...
-	IgnoreError IgnoreErrorFunc
-	// Multiplex returns re-purposed meta clients. The sdk will execute the table with each of them
-	Multiplex func(meta ClientMeta) []ClientMeta
-	// DeleteFilter returns a list of key/value pairs to add when truncating this table's data from the database.
-	DeleteFilter func(meta ClientMeta, parent *Resource) []interface{}
-	// Post resource resolver is called after all columns have been resolved, and before resource is inserted to database.
-	PostResourceResolver RowResolver
-	// Options allow modification of how the table is defined when created
-	Options TableCreationOptions
+ // Name of table
```
Comment (erezrokah) [inline docs/developers/sdk/table/overview.md:7]: Will check if this looks ok on the preview URL

File: docs/cli/policy/testing.md [hunk 2]
```diff
@@ -2,19 +2,17 @@
 
 In order to continue to rely on policies we need to ensure that we are able to handle edge cases and make sure any changes do not result in a regression. To do that the most common way is to have a suite of test cases and ensure the output matches the expected output for specific inputs.
 
-### High level overview of Policy Testing:
+## High level overview of Policy Testing
```
Comment (erezrokah) [inline docs/cli/policy/testing.md:5]: Headers hierarchy should be `H1->H2` and not `H1->H3`

File: docs/developers/cq-gen.md [hunk 3]
```diff
@@ -19,14 +19,13 @@ go install github.com/cloudquery/cq-gen
...
 
-
 ```go
 //go:build tools
 
 package tools
 
 import (
-	_ "github.com/cloudquery/cq-gen"
+ _ "github.com/cloudquery/cq-gen"
```
Comment (erezrokah) [inline docs/developers/cq-gen.md:28]: The code block changes don't look good, I'l revert those
